docs/learn/documentation/versioned/api/samza-sql.md

```diff
@@ -19,65 +19,57 @@ title: Samza SQL
+### Code Examples
 
-### Running Samza SQL on your local machine
-The [Samza SQL console](https://samza.apache.org/learn/tutorials/0.14/samza-tools.html) allows you to experiment with Samza SQL locally on your machine. 
+The [Hello Samza](https://github.com/apache/samza-hello-samza/tree/master/src/main/java/samza/examples/sql) 
+SQL examples demonstrate how to use Samza SQL API.  
 
-#### Setup Kafka
-Follow the instructions from the [Kafka quickstart](http://kafka.apache.org/quickstart) to start the zookeeper and Kafka server.
+- The [sampling](https://github.com/apache/samza-hello-samza/tree/master/src/main/java/samza/examples/sql/samza-sql-samples-yarn)
+is a simple sampling job.
 
-Let us create a Kafka topic named “ProfileChangeStream” for this demo.
+- The [filter](https://github.com/apache/samza-hello-samza/tree/master/src/main/java/samza/examples/sql/samza-sql-filter) 
+demonstrates filtering and insert Samza SQL job.
 
-```bash
->./deploy/kafka/bin/kafka-topics.sh --create --zookeeper localhost:2181 --replication-factor 1 --partitions 1 --topic ProfileChangeStream
-```
-
-Download the Samza tools package from [here](https://samza.apache.org/learn/tutorials/0.14/samza-tools.html) and use the `generate-kafka-events` script populate the stream with sample data.
+- The [group by](https://github.com/apache/samza-hello-samza/tree/master/src/main/java/samza/examples/sql/samza-sql-groupby)
+show how to do group by.
 
-```bash
-> cd samza-tools-<version>
-> ./scripts/generate-kafka-events.sh -t ProfileChangeStream -e ProfileChange
-```
+- The [join](https://github.com/apache/samza-hello-samza/tree/master/src/main/java/samza/examples/sql/samza-sql-stream-table-join)
+demonstrates how to peform a stream-table join.
 
-#### Using the Samza SQL Console
+### Key Concepts
 
+Each Samza SQL job consists of one or more Samza SQL statements.
+Each statement represents a single streaming pipeline.
 
-The simplest SQL query is to read all events from a Kafka topic `ProfileChangeStream` and print them to the console.
+#### SQL Representation
 
-```bash
-> ./scripts/samza-sql-console.sh --sql "insert into log.consoleoutput select * from kafka.ProfileChangeStream"
-```
-
-Next, let us project a few fields from the input stream.
-
-```bash
-> ./scripts/samza-sql-console.sh --sql "insert into log.consoleoutput select Name, OldCompany, NewCompany from kafka.ProfileChangeStream"
-```
+Support for SQL internally uses Apache Calcite, which provides 
+SQL parsing and query planning. The query is automatically translated 
+to Samza's high level API and runs on Samza's execution engine.
 
-You can also filter messages in the input stream based on some predicate. In this example, we filter profiles currently working at LinkedIn, whose previous employer matches the regex `.*soft`. The function `RegexMatch(regex, company)` is an example of 
-a UDF that defines a predicate. 
+The mapping from SQL to the Samza's high level API is a simple 
```

Comment:
worth having a table of supported operations. what do you think?

https://docs.confluent.io/current/ksql/docs/developer-guide/join-streams-and-tables.html